buildSrc/src/main/kotlin/com/uber/nanoscope/release/GitHubRelease.kt

```diff
@@ -173,55 +171,10 @@ enum class IncrementType {
     MAJOR, MINOR, PATCH
```

Comment:
`internal` for consistency?

Reply:
This was public before so it could be accessed from the downstream build.gradle file

Reply:
Oh got it that makes sense.